### PR TITLE
[TR] PIM-4878: add Yaml wirter

### PR DESCRIPTION
**Pull request by solivier**
| Q                  | A
| ------------------ | ---
| Migration script?  | n 
| Behats             | y
| Specs & Static     | y
| Changelog updated  | n

http://core-ci.akeneo.com/job/pim-pr-behat/3366

### CODE REVIEW

File: src/Pim/Component/Connector/Writer/File/YamlWriter.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * @author    Olivier Soulet <olivier.soulet@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class YamlWriter extends FileWriter
+{
+    const INLINE_ARRAY_LEVEL = 8;
+
+    /** @var string */
+    protected $filePath = '/tmp/export_%datetime%.yml';
```
Comment (jjanvier) [inline src/Pim/Component/Connector/Writer/File/YamlWriter.php]: do we always have that ? would be better to use sys_get_temp_dir() and DIRECTORY_SEPARATOR

**Comment by jjanvier** [comment]
commented, GTM otherwise

File: src/Pim/Component/Connector/Writer/File/YamlWriter.php (hunk 1 above)
Comment (sumbobyboys) [inline src/Pim/Component/Connector/Writer/File/YamlWriter.php]: :+1:

File: src/Pim/Component/Connector/Writer/File/YamlWriter.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+     * {@inheritdoc}
+     */
+    public function write(array $items)
+    {
+        $path = $this->getPath();
+        if (!is_dir(dirname($path))) {
+            mkdir(dirname($path), 0777, true);
+        }
+
+        $yaml = Yaml::dump($items[0], self::INLINE_ARRAY_LEVEL);
```
Comment (sumbobyboys) [inline src/Pim/Component/Connector/Writer/File/YamlWriter.php]: why only the first item ?

File: src/Pim/Component/Connector/Writer/File/YamlWriter.php (hunk 1 above)
Comment (solivier) [inline src/Pim/Component/Connector/Writer/File/YamlWriter.php]: It comes from the FileWriter so yes we always have that.

**Comment by sumbobyboys** [comment]
Good job dude ! GTM :)

**Pull request merged**